CELL TD/Assets/Scripts/UI/EnemyCounter.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class EnemyCounter : MonoBehaviour
+{
+    GameObject[] enemies;
+    public Text enemyCountText;
+
+    void Update()
+    {
+        enemies = GameObject.FindGameObjectsWithTag("Enemy");
```

Comment:
In an updated build we have something that already keeps track of how many enemies there are. Use that instead of this.